datasource/src/main/resources/liquibase/properties.xml

```diff
@@ -8,5 +8,6 @@
     <property name="32_length_text.type" value="VARCHAR(32)" dbms="postgresql"/>
     <property name="64_length_text.type" value="VARCHAR(64)" dbms="postgresql"/>
     <property name="binary_data.type" value="BYTEA" dbms="postgresql"/>
+    <property name="user_active.type" value="boolean" dbms="postgresql"/>
```

Comment:
I assume @srdkr meant that you should create a property with a name `bool` or something like that.
@srdkr, am I right?

Reply:
Yep. Btw, I've added boolean type in my pull request as well:
https://github.com/EpamLifeSciencesTeam/cmwl_pipeline/pull/22/files#diff-b8d310bad3f4e408a1da6a8200aa7d35

Reply:
So guess I can remove it?

Reply:
not sure whether my PR will be merged first, so it makes sense to copy-paste that line =)